plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -101,7 +88,7 @@ def wait_for_resources_to_complete(release_docs, timeout)
               @output.puts "TIMEOUT, pods took too long to get live"
               return false, statuses
             end
-          elsif ready_statuses.all? { |s| s.pod.completed? }
+          elsif ready_statuses.all?(&:stop)
```

Comment:
what does stop mean? can you use something more descriptive?

Reply:
changed to `finished`